src/Zoom.vala

```diff
@@ -99,6 +99,20 @@ public class Gala.Zoom : Object {
         gesture_tracker.connect_handlers (null, (owned) on_animation_update, null);
     }
 
+    private inline Graphene.Point compute_new_pivot_point () {
+        unowned var wins = wm.ui_group;
+        unowned Meta.CursorTracker cursor_tracker = wm.get_display ().get_cursor_tracker ();
+        Graphene.Point coords;
+        Clutter.ModifierType mods;
+        cursor_tracker.get_pointer (out coords, out mods);
```

Comment:
```suggestion
        cursor_tracker.get_pointer (out coords, null);
```

Reply:
After looking at Mutter source code, that's indeed possible, opened https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2951